src/core/soci_backends_config.h.in

```diff
@@ -7,6 +7,6 @@
 #ifndef SOCI_BACKENDS_CONFIG_H
 #define SOCI_BACKENDS_CONFIG_H
 
-#define DEFAULT_BACKENDS_PATH "@CMAKE_INSTALL_PREFIX@/@LIBDIR@"
+#define DEFAULT_BACKENDS_PATH "@CMAKE_INSTALL_LIBDIR@"
```

Comment:
`CMAKE_INSTALL_LIBDIR` is usually supposed to be a relative path (though it can sometimes be absolute if the user specifies it this way.) The old value had `${CMAKE_INSTALL_PREFIX}/${LIBDIR}`, which would be an absolute path. Is it OK for `DEFAULT_BACKENDS_PATH` to be relative?

Reply:
It can be `.`, so it definitely can be relative, yes.

Reply:
Alright then, looks good to me.

Reply:
Actually, taking a closer look at the SOCI code, it looks like the contents of `DEFAULT_BACKENDS_PATH` ultimately get passed to `DLOPEN()`, which would search relative to the current directory in the case of a relative path. In the case of `CMAKE_INSTALL_LIBDIR`, this is not what we want. I think it would be better to set it to `@CMAKE_INSTALL_PREFIX@/@CMAKE_INSTALL_LIBDIR@`.

Reply:
`@CMAKE_INSTALL_PREFIX@/@CMAKE_INSTALL_LIBDIR` won't work if CMAKE_INSTALL_PREFIX is absolute which it is in Nix. Then we'd need to bring back LIBDIR and just statically set it to `LIB`.

Reply:
Why can't `CMAKE_INSTALL_PREFIX` be absolute?

Reply:
> Mabye CMAKE_INSTALL_FULL_LIBDIR can be used instead?

Yes, I think that would be appropriate.